blueprints/showbie-tooling/files/.pre-commit-config.yaml

```diff
@@ -1,24 +1,25 @@
 # see http://pre-commit.com/#usage and run `pre-commit --help` for more info
 # To update these hooks, run `pre-commit autoupdate` and commit the changes.
+repos:
 -   repo: git@github.com:pre-commit/pre-commit-hooks
-    sha: e626cd57090d8df0be21e4df0f4e55cc3511d6ab
+    sha: v1.1.1
     hooks:
     -   id: trailing-whitespace
     -   id: end-of-file-fixer
     -   id: check-yaml
     -   id: check-json
     -   id: check-merge-conflict
 -   repo: git@github.com:pre-commit/mirrors-eslint
-    sha: v3.14.0
+    sha: v4.10.0
     hooks:
     -   id: eslint
         additional_dependencies:
         - eslint
-        - eslint-plugin-ember-suave
+        - eslint-plugin-ember
         args:
         - --quiet
         language_version: 6.9.4
 -   repo: git@github.com:pre-commit/mirrors-scss-lint
-    sha: v0.52.0
+    sha: v0.55.0
```

Comment:
We'll probably switch to stylelint before any of the consuming applications update but I'm pretty sure that SMACSS ordering issue started in v0.53.

Reply:
Ah yeah, I think that was added after running `pre-commit autoupdate` and I figured the stylelint PR would take care of it. 😄